Tests/ASDisplayNodeTests.mm

```diff
@@ -1924,16 +1925,11 @@ - (void)checkBackgroundColorOpaqueRelationshipWithViewLoaded:(BOOL)loaded layerB
   XCTAssertTrue(node.opaque, @"Node should start opaque");
   XCTAssertTrue(node.layer.opaque, @"Node should start opaque");
 
-  node.backgroundColor = [UIColor clearColor];
+  node.backgroundColor = [UIColor blackColor];
```

Comment:
Could you elaborate why this test is failing now?

Reply:
This was just for testing to see if `clearColor` was changing the interpretation of `isOpaque` on `CALayer`